npm-search/assets/css/base.css

```diff
@@ -5,6 +5,11 @@ body {
   margin: 0; 
 } 
 
+h1 {
+  -webkit-margin-before: 0.3em;
```

Comment:
yeah removing it and bringing the text to a single line centrally aligned with the image